.gitignore

```diff
@@ -4,6 +4,27 @@ tweet_data/
 data/
 venv/
 .vscode/
+<<<<<<< HEAD
```

Comment:
as mentioned in the comment this needs fixing, you can restore it to the original by doing `git checkout HEAD^1 -- .gitignore` (or something like that)

Reply:
I think this is changed now